commit 36dbd3287fefdfceeb6a82997241c26df29882c8
Author: Chao Yu <chao@kernel.org>
Date:   Mon Sep 26 19:45:05 2016 +0800

    f2fs: do fault injection initialization in default_options
    
    Do fault injection initialization in default_options to keep consistent
    with other default option configurating.
    
    Signed-off-by: Chao Yu <yuchao0@huawei.com>
    Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>

diff --git a/fs/f2fs/super.c b/fs/f2fs/super.c
index a06eee4a8fa1..785d0fdf6e76 100644
--- a/fs/f2fs/super.c
+++ b/fs/f2fs/super.c
@@ -363,10 +363,6 @@ static int parse_options(struct super_block *sb, char *options)
 	char *p, *name;
 	int arg = 0;
 
-#ifdef CONFIG_F2FS_FAULT_INJECTION
-	f2fs_build_fault_attr(sbi, 0);
-#endif
-
 	if (!options)
 		return 0;
 
@@ -990,6 +986,10 @@ static void default_options(struct f2fs_sb_info *sbi)
 #ifdef CONFIG_F2FS_FS_POSIX_ACL
 	set_opt(sbi, POSIX_ACL);
 #endif
+
+#ifdef CONFIG_F2FS_FAULT_INJECTION
+	f2fs_build_fault_attr(sbi, 0);
+#endif
 }
 
 static int f2fs_remount(struct super_block *sb, int *flags, char *data)